cli/ops/os.rs

```diff
@@ -41,11 +41,15 @@ fn op_start(
   _zero_copy: Option<PinnedBuf>,
 ) -> Result<JsonOp, ErrBox> {
   let gs = &state.global_state;
-
+  let script_args = if gs.flags.argv.len() >= 2 {
+    gs.flags.argv.clone().split_off(2)
+  } else {
```

Comment:
I suspect this might not be working properly in REPL. `flags.argv` should be refactored as well, to separately handle `<script>` and `<sciprt_args>` on Rust side.

Reply:
This conditional is to support the REPL. But I agree that flags.argv should be refactored.